### PR TITLE
Update Grafana Agent other install topic

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description

Update Grafana Agent Other systems install steps
 
* Clean up and reformat install steps
* Add in missing information for different OSes

#### Notes to the Reviewer

TODO: 

1. How do you run Agent in Windows if you download the binary vs installer.
1. When you run in Linux... is this just stand-alone and not a systemd service? How can  you set up Linux to launch on boot (like a systemd service)? 
1. What about macOS.. anything different here?

#### PR Checklist

- [ ] CHANGELOG updated
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [inline docs/sources/static/set-up/install-agent-binary.md:30]:
```suggestion
   chmod +x EXTRACTED_BINARY
```

**Comment by clayton-cornell** [comment]
@grafana/grafana-agent-maintainers There's a couple of open questions in this topic.... marked as ToDo

TODO:

1. How do you run Agent in Windows if you download the binary vs installer.
2. When you run in Linux... is this just stand-alone and not a systemd service? How can you set up Linux to launch on boot (like a systemd service)?
3. What about macOS.. anything different here?

File: docs/sources/static/set-up/install-agent-binary.md
Comment (rfratto) [inline docs/sources/static/set-up/install-agent-binary.md:10]: Did you mean to say Flow here? This is the static mode install instructions.

File: docs/sources/static/set-up/install-agent-binary.md
Comment (rfratto) [inline docs/sources/static/set-up/install-agent-binary.md:8]: Will it be confusing if we don't mention what mode this is here? (static mode vs static mode operator or Flow mode)

File: docs/sources/static/set-up/install-agent-binary.md
Comment (rfratto) [inline docs/sources/static/set-up/install-agent-binary.md:23]:
```suggestion
1. Navigate to the current Grafana Agent [releases](https://github.com/grafana/agent/releases).
```

File: docs/sources/static/set-up/install-agent-binary.md
Comment (rfratto) [inline docs/sources/static/set-up/install-agent-binary.md:27]: Do we need to explain why the user needs to run this, or should it be fine to tell them to copy-paste as-is?

File: docs/sources/static/set-up/install-agent-binary.md
Comment (rfratto) [inline docs/sources/static/set-up/install-agent-binary.md:45]: It's just standalone; we only support systemd services through our DEB and RPM system packages.

Same with macOS: this page instructs you to build a standalone service; if you want it as a system service on macOS, you should follow the bespoke macOS instructions to install the Homebrew formula.

File: docs/sources/static/set-up/install-agent-binary.md
Comment (rfratto) [inline docs/sources/static/set-up/install-agent-binary.md:49]: It's the same:

```
PATH_TO_BINARY -config.file CONFIG_FILE
```

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [reply docs/sources/static/set-up/install-agent-binary.md:10]: Copy/paste error, I'll fix it

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [reply docs/sources/static/set-up/install-agent-binary.md]: It works either way. The topic is in the static mode section... but it's easy to add static mode to the heading and this removes any question

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [reply docs/sources/static/set-up/install-agent-binary.md]: Fixed

File: docs/sources/static/set-up/install-agent-binary.md
Comment (rfratto) [reply docs/sources/static/set-up/install-agent-binary.md]: Yeah, that's true, I'm considering that the other install pages currently explicitly state what mode they're for, and whether that has an impact on SEO when trying to find the correct docs from Google.

I'll leave the final call up to you though, you definitely know more about this than I do :)

Comment (clayton-cornell) [reply docs/sources/static/set-up/install-agent-binary.md]: fixed

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [reply docs/sources/static/set-up/install-agent-binary.md]: Updated the text a bit. I think this gets closer to what we want to say now.

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [reply docs/sources/static/set-up/install-agent-binary.md]: fixed

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [inline docs/sources/static/set-up/install-agent-binary.md:22]:
```suggestion
* [Linux]({{< relref "./install-agent-linux.md" >}})
* [macOS]({{< relref "./install-agent-macos.md" >}})
* [Windows]({{< relref "./install-agent-on-windows.md" >}})
```

**Review by rfratto — APPROVED** [review]

File: docs/sources/static/set-up/install-agent-binary.md
Comment (clayton-cornell) [inline docs/sources/static/set-up/install-agent-binary.md:48]:
```suggestion
   EXTRACTED_BINARY -config.file CONFIG_FILE 
```

**Pull request merged**
